GreetingTest.java

```diff
@@ -16,13 +16,13 @@ public void setUp() {
 
     @After
     public void tearDown() {
-        System.setOut(null);
+        PrintStream originalOut = System.out;
```

Comment:
But isn't this Ssytem.out the thing you changed it to in the setup? The originalOut should be a test class field set in setUp before the change and then restored here. Almost there!